static/js/publisher/components/RegisteredSnaps/RegisteredSnaps.tsx

```diff
@@ -89,9 +91,9 @@ function RegisteredSnaps({
               </span>
             ) : (
               <button className="p-button--link u-no-margin--bottom">
```

Comment:
I know it's not introduced in this PR, but seems odd to have a `button` with a link inside.

Not sure what was original intention here, but if functionally it's a link (to a separate view), and we want it to look like a link (`p-button--link` suggests that), it can just be a link. The wrapping button may be redundant and can possibly be removed?